pkgs/types/src/lib.zig

```diff
@@ -33,16 +49,22 @@ pub const RootHex = utils.RootHex;
 pub const ZERO_HASH = utils.ZERO_HASH;
 pub const ZERO_HASH_4000 = utils.ZERO_HASH_4000;
 pub const StateTransitionError = utils.StateTransitionError;
-pub const HistoricalBlockHashes = utils.HistoricalBlockHashes;
-pub const JustifiedSlots = utils.JustifiedSlots;
-pub const JustificationsRoots = utils.JustificationsRoots;
-pub const JustificationsValidators = utils.JustificationsValidators;
 pub const BytesToHex = utils.BytesToHex;
+pub const Bytes52 = utils.Bytes52;
 pub const GenesisSpec = utils.GenesisSpec;
 pub const ChainSpec = utils.ChainSpec;
 pub const sszClone = utils.sszClone;
+pub const IsJustifiableSlot = utils.IsJustifiableSlot;
 
 const zk = @import("./zk.zig");
 pub const ZkVm = zk.ZkVm;
 pub const BeamSTFProof = zk.BeamSTFProof;
 pub const BeamSTFProverInput = zk.BeamSTFProverInput;
+
+// TODO: use refAllDeclsRecursive instead of refAllDecls
+// Avoids refAllDeclsRecursive as ssz lists tests fail
+// SignedAttestation and Validator both have byte array fields
+// sszlibrary should use std.meta.eql instead of std.mem.eql for struct types
```

Comment:
@g11tech  @noopur23 can we have a separate issue for this as well, I don't want to wait for ssz PR to get merged and have a release before we can get this PR merged